aws-dynamodb-point-in-time-recovery-disabled.yaml

```
@@ -0,0 +1,37 @@
+rules:
+- id: aws-dynamodb-point-in-time-recovery-disabled
+  patterns:
+  - pattern: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+      }
+  - pattern-not-inside: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+        point_in_time_recovery {
+          enabled = true
+        }
```

Could this have other fields? Maybe we can add ellipses to make this rule more future proof
```suggestion
        point_in_time_recovery {
          ...
          enabled = true
          ...
        }
```

---

No, it can't have any other fields currently, but we can add ellipses to future-proof it, as you say!